src/templates/index.hbs

```diff
@@ -232,6 +244,73 @@
       </div>
     </section>
 
+<section id="podcast" class="container content-section text-center">
```

Comment:
I'd reindent it so it matches the file's indentation patterns.

Reply:
That's ok, I think what errors are because i had some conflicts and maybe i accepted the wrong file, I will try to resolve all comments in the week

Reply:
The indentation still isn't matching the file's pattern, but this is a minor issue, feel free to merge.

Reply:
Sorry for this, I thought about using automatic formating, but I I was afraid haha and forget of the manually indent

Reply:
Sorry for this, I thought about using automatic formating, but I I was afraid haha and forget of the manually indent. Do you want to fix this?